src/components/TextInput/TextInput.tsx

```diff
@@ -110,14 +110,16 @@ export interface Props {
    * The default value of the input
    */
   defaultValue?: string | number;
-}
+};
 
 /**
  * BETA: This component is still a work in progress and is subject to change.
  *
  * ```ts
  * import {TextInput} from "@chanzuckerberg/eds";
  * ```
+ *
+ * Text input component for one line of text. For multiple lines, consider the Textarea upcoming-component.
```

Comment:
I'd prefer it if we didn't specify if some other component is in the `upcoming-components` directory because, when we move that component over, we're probably not going to remember to come back and update this comment.

Reply:
Ah that makes sense, would changing it to just `consider the Textarea component` be more future proof?

Reply:
Yeah that's what I'm thinking; just drop the `upcoming-` part 👍